doc/classes/Signal.xml

```diff
@@ -4,6 +4,8 @@
 		Class representing a signal defined in an object.
 	</brief_description>
 	<description>
+		Signals can be connected to [Callable]s and emitted. When a signal is emitted, are connected callables are called.
```

Comment:
```suggestion
		Signals can be connected to [Callable]s and emitted. When a signal is emitted, all connected callables are called.
```

Should this say "all"?

Reply:
Yes. It refers to all callables connected to this specific signal.